schemas_src/concrete/MPSectionCategory.json

```diff
@@ -34,6 +34,9 @@
         "pageBreakStyle": {
           "type": "number"
         },
+        "groupID": {
+          "type": "string"
```

Comment:
Hello @asouqi 
Shouldn't this be an array of some sort?

Reply:
no, will be just a string that holds id one of the types will add in `manuscripts-data` it could be like this:  `MPSectionCategory:abstracts`, `MPSectionCategory:body`, `MPSectionCategory:backmatter`

Reply:
updated to array